managers/redis/manager.go

```diff
@@ -0,0 +1,237 @@
+	objName objectType, objKey []string,
+) ([]T, error) {
+	redisCmd := redisClient.MGet(ctx, objKey...)
+	if redisCmd.Err() != nil {
+		// todo(UMV): print when this will be done https://github.com/Wissance/stringFormatter/issues/14
+		logger.Warn(sf.Format("An error occurred during fetching {0}: from Redis server", objName))
+		return nil, redisCmd.Err()
+	}
+
+	raw := redisCmd.Val()
+	result := make([]T, len(raw))
+	var unMarshalledRaw interface{}
+	for i, v := range raw {
+		err := json.Unmarshal([]byte(v.(string)), &unMarshalledRaw)
+		if err != nil {
+			logger.Error(sf.Format("An error occurred during unmarshall {0} : \"{1}\"", objName, objKey))
+			return nil, fmt.Errorf("json.Unmarshal failed: %w", err)
+		}
+		result[i] = unMarshalledRaw.(T)
+	}
+	return result, nil
+}
+
+// this functions gets object that stored as a LIST Object type
+func getObjectsListFromRedis[T any](redisClient *redis.Client, ctx context.Context, logger *logging.AppLogger,
+	objName objectType, objKey string,
+) ([]T, error) {
+	redisCmd := redisClient.LRange(ctx, objKey, 0, -1)
+	if redisCmd.Err() != nil {
+		logger.Warn(sf.Format("An error occurred during fetching {0}: \"{1}\" from Redis server", objName, objKey))
+		return nil, redisCmd.Err()
+	}
+
+	// var obj T
+	items := redisCmd.Val()
+	if len(items) == 0 {
+		logger.Warn(sf.Format("Received zero list items {0}: \"{1}\" from Redis server", objName, objKey))
+		return nil, errors.ErrZeroLength
+	}
+	var result []T
+	var portion []T
+	for _, rawVal := range items {
+		jsonBin := []byte(rawVal)
+		err := json.Unmarshal(jsonBin, &portion) // already contains all SLICE in one object
+		if err != nil {
+			logger.Error(sf.Format("An error occurred during unmarshall {0} : \"{1}\"", objName, objKey))
+			return nil, fmt.Errorf("json.Unmarshal failed: %w", err)
+		}
+		result = append(result, portion...)
+	}
+	return result, nil
+}
+
+// If such a key exists, the value will be overwritten without error
+func setString(redisClient *redis.Client, ctx context.Context, logger *logging.AppLogger,
+	objName objectType, objKey string, objValue string,
+) error {
+	statusCmd := redisClient.Set(ctx, objKey, objValue, 0)
+	if statusCmd.Err() != nil {
+		logger.Warn(sf.Format("An error occurred during Set {0}: \"{1}\" from Redis server", objName, objKey))
+		return statusCmd.Err()
+	}
+	return nil
+}
+
+// Returns an error, if 0 items are deleted
+func delKey(redisClient *redis.Client, ctx context.Context, logger *logging.AppLogger, objName objectType, objKey string) error {
+	redisIntCmd := redisClient.Del(ctx, objKey)
+	res := redisIntCmd.Val()
+	if res == 0 {
+		logger.Warn(sf.Format("An error occurred during Del, 0 items deleted {0}: \"{1}\" from Redis server", objName, objKey))
+		return errors.ErrNotExists
+	}
+	return nil
+}
+
+// Adds a row to the list
+func rPushString(redisClient *redis.Client, ctx context.Context, logger *logging.AppLogger, objName objectType, objKey string, objValue string) error {
```

Comment:
I suggest a new new - `appendStringToRedisList`

Reply:
All private method should be provided with a commentaries in same styling as for all **TOO**
```
// FuncName - short description
/* More detailed description
 * Arguments:
 *    - arg0
 *   ...
 *   -argN
 * Returns: describe what it returns
 */
```